src/core/observable.js

```diff
@@ -128,7 +128,7 @@ class SafeSubscriber extends Subscriber {
     try {
       fn.call(this._context, value);
     } catch (err) {
-      this.unsubscribe();
+      this.error(err);
```

Comment:
Makes sense to throw an error here. But are there certain cases where we should still unsubscribe, or is this now something the developer should handle completely?

Reply:
It will still unsubscribe after the error callback is called.